lib/mongoid.rb

```diff
@@ -95,11 +95,21 @@ def default_client
   # @example Disconnect all active clients.
   #   Mongoid.disconnect_clients
   #
-  # @return [ true ] True.
+  # @return [ Array<Mongo::Client> ] The driver clients.
   def disconnect_clients
     Clients.disconnect
   end
 
+  # Reconnect all active clients.
+  #
+  # @example Reconnect all active clients.
+  #   Mongoid.reconnect_clients
+  #
+  # @return [ Array<Mongo::Client> ] The driver clients.
```

Comment:
Hey @johnnyshields -- here and elsewhere, do you think there's value in actually returning the list of clients? I don't believe the current return value is intentional (or even useful). I think I'd rather leave the documented return value as `true` (or at least "truthy"), rather than be committed to always returning a list of clients here.

Reply:
@jamis the documentation should match what the method does, and today the method returns the list of clients. It is certainly fine to raise another PR and change the return value, I doubt anyone is relying on this behavior today.

Reply:
You're not wrong, Johnny. However, it's also fair to say that the method returns truthy, which is what the documentation today (poorly) implies. I much prefer a truthy return value, than telling the world to expect it to return a list of clients. There are other ways to get the list of clients, and I'd really rather this not be one of them.

Reply:
OK so would like me to change this method behavior of both `disconnect` (existing) and `reconnect` (new) in this PR?

Reply:
`reconnect` should mirror the behavior of `disconnect`. I'm honestly more worried about the documentation (and the contract it exposes) than the actual implementation, but as long as we're not promising clients more than we want to deliver (e.g. exposing the client list as a return value here), I'm okay.

Reply:
@jamis this is now done--all methods in this PR return true.